tensorflow/workspace.bzl

```diff
@@ -359,37 +359,45 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
         },
     )
 
+    PROTOBUF_urls =[
```

Comment:
The protobuf release cycle seems to be very slow: https://github.com/protocolbuffers/protobuf/pull/4862#issuecomment-426058251
Can we do something about it instead of waiting for more months?